toolkit/tools/grapher/grapher.go

```diff
@@ -122,7 +123,9 @@ func addUnresolvedPackage(g *pkggraph.PkgGraph, pkgVer *pkgjson.PackageVer) (new
 		return
 	}
 
-	nodes, err := g.FindBestPkgNode(pkgVer)
+	// Double check that the package is not already in the graph. The graph should have already used that node before
+	// this point.
```

Comment:
I understand the first sentence but not really the second one. If I understand the `nodes != nil` check correctly, the node should **NOT** be present in the graph at this point. Could you rephrase the comment a bit?